typescript/sdk/src/providers/SmartProvider/SmartProvider.ts

```diff
@@ -1,4 +1,5 @@
-import { providers } from 'ethers';
+import { BigNumber, providers } from 'ethers';
+import { resolveProperties } from 'ethers/lib/utils.js';
```

Comment:
incorrect import. This will screw up tree shaking and can cause other problems. I would guess that importing `utils` and then calling `utils.resolveProperties` is what you're looking for